src/editor/Collaboration.js

```diff
@@ -3,8 +3,18 @@ define(function (require, exports, module) {
 
     var SimpleWebRTC    = require("simplewebrtc");
 
-    function Collaboration() {
-        var webrtc = new SimpleWebRTC({
+    var _webrtc,
+        _pending,
+        _changing,
+        _room,
+        _codemirror;
+
+    function initialize() {
```

Comment:
Tbh, I don't think they need to be two separate functions. Can't we just have the `Editor` constructor call a single function called `connect()` which has the contents of both `initialize` and `init`?
```js
function connect(codemirror) {
    if(_webrtc) {
        console.error("Collaboration already initialized");
        return;
    }

    _webrtc = ...
    // Rest of this function

    _webrtc.joinRoom(_room, function() {
        // Rest of the code
    });

    _webrtc.connection.on("message", _handleMessage);
}
```